tools/install-deps-rpi.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+#
+##########################################################################
+#
+# Copyright (c) 2017 SKKU ESLAB, and contributors. All rights reserved.
+# 
+# Contributor: Gyeonghwan Hong<redcarrottt@gmail.com>
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+##########################################################################
+# install-deps-raspberry-pi2_3.sh: Script to install dependent packages in
+#                           raspberry pi 2 or 3 target board
+##########################################################################
+
+# print_progress(): Print the progress
+#  - $1(STEP_NUM): (int) Progress step's number
+#  - $2(STEP_DESC): (string) Progress step's description
+print_progress() {
+  STEP_NUM=$1
+  STEP_DESC=$2
+
+  WARN_COLO="\033[31;47m"
+  INFO_COLO="\033[36m"
+  INIT_COLO="\033[0m"
+  
+  echo -e "${INFO_COLO}Step ${STEP_NUM}. ${STEP_DESC} ${INIT_COLO}"
+}
+
+# Step 1. Install packages by apt-get
+print_progress 1 "Install dependent packages..."
+sudo apt-get update
+sudo apt-get -y install make g++-4.8                                          \
+  wiringpi libdbus-1-dev glib-2.0 libdbus-glib-1-2 bison byacc                 \
+  libdbus-glib-1-2-dbg libdbus-glib-1-dev zip sqlite3 libsqlite3-dev cmake    \
+  libgtk2.0-dev pkg-config libavcodec-dev libavformat-dev libswscale-dev git  \
+  python-dev python-numpy libjpeg-dev libpng-dev libtiff-dev libjasper-dev    \
+  libdc1394-22-dev automake libtool libssl-dev libnl-3-dev libnl-genl-3-dev   \
+  python3 udhcpd libv4l-dev libboost-dev libboost-serialization-dev gstreamer1.0
```

Comment:
Add ```libopencv-dev```

Reply:
I think this should be done in another commit, as you pulled request in #26 .
My PR is just about the test system. please do the modification in your PR.

Reply:
Okay. I will do.